src/app/qgsmaptooladdcircularstring.cpp

```diff
@@ -83,6 +86,16 @@ void QgsMapToolAddCircularString::keyPressEvent( QKeyEvent* e )
 
     createCenterPointRubberBand();
   }
+
+  if ( e && e->key() == Qt::Key_Escape )
+  {
+    mPoints.clear();
+    delete mRubberBand; mRubberBand = nullptr;
```

Comment:
I cut-copy-pasted, but it surely is a valid point. I will add this and the others to the _code cleanup and improved readability in circular maptools_ commit on monday.

Would this be a point for the _qgisstyle_ tool? It inserts missing newlines in other situations:
```c++
if ( expression )
{
  doSomething(); }
```